src/devices/moes.ts

```diff
@@ -597,6 +597,104 @@ const definitions: DefinitionWithExtend[] = [
+                .withUnit('°C')
+                .withDescription('Low protect temperature')
+                .withValueMin(0)
+                .withValueMax(10),
+            //backlight_brightness
+            new exposes.Numeric('backlight_brightness', exposes.access.STATE_SET)
+                .withUnit('%')
+                .withDescription('Backlight brightness')
+                .withValueMin(0)
+                .withValueMax(100),
+            //Screen time
+            new exposes.Enum(
+                'screen_time',
+                exposes.access.STATE_SET,
+                [10, 20, 30, 40, 50, 60].map((secs) => `${secs} seconds`),
+            )
+                .withDescription('Screen on time')
+                .withCategory('config'),
+            e.binary('rgblight', exposes.access.STATE, true, false),
+        ],
+        meta: {
+            tuyaDatapoints: [
+                [1, 'system_mode', tuya.valueConverterBasic.lookup({off: false, heat: true})],
+                [2, 'preset', tuya.valueConverterBasic.lookup({Manual: 0, 'Temporary manual': 1, Program: 2, Eco: 3})],
+                [16, 'local_temperature', tuya.valueConverter.divideBy10],
+                [18, 'min_temperature_limit', tuya.valueConverter.divideBy10],
+                [32, 'sensor', tuya.valueConverterBasic.lookup({'Internal sensor (IN)': 0, 'Both (AL)': 1, 'External sensor (OU)': 2})],
+                [34, 'max_temperature_limit', tuya.valueConverter.divideBy10],
+                [39, 'child_lock', tuya.valueConverter.lockUnlock],
+                [47, 'running_state', tuya.valueConverterBasic.lookup({heat: 0, idle: 1})],
+                [48, 'backlight_brightness', tuya.valueConverter.raw],
+                [50, 'current_heating_setpoint', tuya.valueConverter.divideBy10],
+                [101, 'local_temperature_calibration', tuya.valueConverter.raw],
+                [102, 'week_program_l3_1', tuya.valueConverter.raw],
+                [103, 'week_program_l3_2', tuya.valueConverter.raw],
+                [104, 'week_program_l3_3', tuya.valueConverter.raw],
+                [105, 'week_program_l3_4', tuya.valueConverter.raw],
+                [106, 'week_program_l3_5', tuya.valueConverter.raw],
+                [107, 'week_program_l3_6', tuya.valueConverter.raw],
+                [108, 'week_program_l3_7', tuya.valueConverter.raw],
+                [109, 'floor_temperature', tuya.valueConverter.divideBy10],
+                [110, 'deadzone_temperature', tuya.valueConverter.divideBy10],
+                [111, 'high_protect_temperature', tuya.valueConverter.divideBy10],
+                [112, 'low_protect_temperature', tuya.valueConverter.divideBy10],
+                [113, 'eco_temperature', tuya.valueConverter.divideBy10],
+                [
+                    114,
+                    'screen_time',
+                    tuya.valueConverterBasic.lookup({
+                        '10 seconds': 0,
```

Comment:
```suggestion
                        '10_sec': 0,
```

(do the same for all others values)

Reply:
That will make the selector show `10_sec` instead of `10 seconds`. Why not keep it more human readable?

Reply:
It's the convention for every enum value, I'm also fine with `10_seconds`

Reply:
Done